src/mlui/enums/Adabound.py

```diff
@@ -0,0 +1,225 @@
+
+
+
```

Comment:
Unnecessary spaces. Please use the [black](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter) formatter to ensure code formatting is consistent and neat.

Reply:
solved